src/modules/collection/utils.ts

```diff
@@ -135,3 +136,14 @@ export function isTPCollection(collection: Collection): boolean {
 export function getCollectionFactoryContract(chainId: ChainId) {
   return getContract(ContractName.CollectionFactoryV3, chainId)
 }
+
+export const getTPThresholdToReview = (totalItems: number) => {
+  // Reference: https://governance.decentraland.org/proposal/?id=f69c4d40-aaaf-11ec-87a7-6d2a41508231
+  // Max to review, 300 items, min to review is 50 or 1% of the collection
+  if (totalItems! < MIN_TP_ITEMS_TO_REVIEW) {
+    return totalItems
+  } else if (totalItems! > MIN_TP_ITEMS_TO_REVIEW && totalItems! < MAX_TP_ITEMS_TO_REVIEW) {
+    return MIN_TP_ITEMS_TO_REVIEW
+  }
+  return Math.min(Math.floor(totalItems! * TP_TRESHOLD_TO_REVIEW), MAX_TP_ITEMS_TO_REVIEW)
```

Comment:
I think that the whole function can be reduced to:
```suggestion
  return Math.min(Math.max(Math.floor(totalItems! * TP_TRESHOLD_TO_REVIEW), MIN_TP_ITEMS_TO_REVIEW), MAX_TP_ITEMS_TO_REVIEW)
```
That is, if the minimum amount of items to review according to the threshold is less than 50, then review 50 items, but if it's greater than 50, review as many items as 300.